unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs

```diff
@@ -24,30 +30,35 @@ public class LambdasService : ILambdasService
         {
             var postDataJson = JsonUtility.ToJson(postData);
             var url = GetUrl(endPoint, urlEncodedParams);
-            var wr = webRequestController.Ref.Post(url, postDataJson, requestAttemps: attemptsNumber, timeout: timeout);
+            var wr = webRequestController.Ref.Post(url, postDataJson, requestAttemps: attemptsNumber, timeout: timeout, disposeOnCompleted: false);
+            var transaction = urlTransactionMonitor.Ref.TrackWebRequest(wr.asyncOp, endPointTemplate, data: postDataJson, finishTransactionOnWebRequestFinish: false);
 
-            return SendRequestAsync<TResponse>(wr, cancellationToken, endPoint, urlEncodedParams);
+            return SendRequestAsync<TResponse>(wr, cancellationToken, endPoint, transaction, urlEncodedParams);
         }
 
         public UniTask<(TResponse response, bool success)> Get<TResponse>(
+            string endPointTemplate,
             string endPoint,
-            int timeout = ILambdasService.DEFAULT_TIMEOUT,
-            int attemptsNumber = ILambdasService.DEFAULT_ATTEMPTS_NUMBER,
+            int timeout = 30,
+            int attemptsNumber = 3,
```

Comment:
Out of curiosity, why this change?

Reply:
Clean-up changed it for some reason, I will revert

Reply:
Ah no, it was automatic refactoring of parameters by Rider, it screwed the things up , that's why I didn't apply it in the Post method